### PR TITLE
Added an option to set rel=nofollow to the social links

### PR DESCRIPTION
**Pull request by kkomelin**
Please review.
ref #13

### CODE REVIEW

File: socialfield.module [hunk 1]
```diff
@@ -606,6 +607,13 @@ function socialfield_field_formatter_settings_form($field, $instance, $view_mode
     '#default_value'  => $settings['link_text'],
   );
 
+  $element['link_rel_nofollow'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Add rel="nofollow"'),
+    '#description' => t('Each icon link will have "rel" attribute set to "nofollow". This will disallow search engines to follow the link.'),
+    '#default_value'  => $settings['link_rel_nofollow'],
```
Comment (klonos) [inline socialfield.module]: ...double space there.

Comment (kkomelin) [inline socialfield.module]: Thanks @klonos! Fixed here https://github.com/backdrop-contrib/socialfield/pull/14/commits/0679929620528d7a9519379188e288fbc0a6e3df

**Comment by CybersecDan** [comment]
Thanks for the PR @kkomelin. My apologies for the delay. :(

**Comment by kkomelin** [comment]
Thanks @danielrose28 !

**Comment by kkomelin** [comment]
Please also close the corresponding issue https://github.com/backdrop-contrib/socialfield/issues/13

**Comment by jenlampton** [comment]
A note here, I reopened the main issue because I'm not sure about making this a setting. That said, here's no reason to revert this change (yet), we may discuss over there and decide it's perfect as is. I just wanted to hear about a good use case for both when and when-not to use nofollow to be sure a setting was warranted :)

**Comment by jenlampton** [comment]
Also thank you both @kkomelin and @danielrose28 for making a PR and merging it! go team 👍

**Comment by danielrose28** [comment]
Wish I had more of a case for it either way. I think it's useful, but honestly can't think of a reason for it not to be. I know what it's for but haven't done enough with it to give pros/cons, etc. Feel free to school me on this guys. :)

**Comment by jenlampton** [comment]
Some light reading:
https://moz.com/community/q/is-it-better-nofollow-or-follow-links-to-external-social-pages
https://beymour.com/when-you-should-and-shouldnt-use-a-nofollow
http://sixrevisions.com/html/rel-nofollow/

**Comment by danielrose28** [comment]
Thanks @jenlampton,  I'll definitely check these out. :)

**Comment by kkomelin** [comment]
Hi @jenlampton,

I implemented the rel=nofollow as an option because I always try to take into account and respect current users's interests. I can't just turn on/off a feature that can potentially negatively affect their SEO rank (even indirectly). That's why rel=nofollow is turned off by default, and it doesn't change anything for existing users.

However, less number of choices leads to better usability and I believe users appreciate it in Backdrop. My only consideration is that any improvement should not harm existing users. 

On top of that, as a Backdrop founder, you might have own vision, so feel free to decide. I will support your decision.

**Comment by jenlampton** [comment]
@kkomelin I copied your comment and added a reply over in https://github.com/backdrop-contrib/socialfield/issues/13. 
@danielrose28 I moved your/my previous comments over there too (and deleted them from here) ;)

(the issues are where we usually talk about why/how/if - and the PR is where we do a code review)

**Comment by jenlampton** [comment]
@danielrose28 I moved your comment over there too ;)

**Comment by jenlampton** [comment]
I've reverted the setting in favor of a theme-based approach to this problem, and proposed https://github.com/backdrop-contrib/socialfield/pull/17 as an alternative.

**Comment by CybersecDan** [comment]
Makes sense. Sorry for just simply doing the merge. For future PRs I'll try to be more thorough.

**Comment by jenlampton** [comment]
No need to apologize!  Thank you for paying attention to the issue faster than I could :)

**Comment by kkomelin** [comment]
@danielrose28 I still believe you have made a correct decision by merging my PR, so thanks. @jenlampton can have own opinion and I'm Okay with that.

**Pull request merged**